recipe_scrapers/keukenliefdenl.py

```diff
@@ -0,0 +1,58 @@
+# mypy: allow-untyped-defs
+
+from ._abstract import AbstractScraper
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class KeukenLiefdeNL(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "keukenliefde.nl"
+
+    def author(self):
+        return self.soup.find("meta", {"name": "author"}).get("content")
+
+    def title(self):
+        return self.soup.find("meta", {"property": "og:title"}).get("content")
+
+    def category(self):
+        return self.soup.find(
+            "div", {"class": "article-meta-item sp gerecht"}
+        ).getText()
+
+    def total_time(self):
+        return get_minutes(
+            self.soup.find("div", {"class": "article-meta-item sp tijd"}).getText()
+        )
+
+    def yields(self):
+        return get_yields(
+            self.soup.find("div", {"class": "article-meta-item sp aantal"}).getText()
+        )
+
+    def image(self):
+        return self.schema.image()
+
+    def ingredients(self):
+        ingredients = self.soup.find(
+            "div", {"id": "clipboard-ingredients"}
+        ).findChildren("li")
+
+        return [normalize_string(i.get_text()) for i in ingredients]
+
+    def instructions(self):
+        # Old recipes are written in paragraphs, new ones are a list.
+        paragraphs = self.soup.find("div", {"class": "preparation"}).findChildren("p")
+        listitems = self.soup.find("div", {"class": "preparation"}).findChildren("li")
```

Comment:
A mix of guidance about our 'house style' for BeautifulSoup method calls, and also the [upstream documentation about some method renaming over time](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#find-all):

We should generally use Pythonic [snake-case](https://en.wikipedia.org/wiki/Snake_case) for these method calls - so `getText` becomes `get_text`.

In the case of `findChildren` - I think that `find_all` should be a valid replacement in these cases.

```suggestion
        return get_minutes(
            self.soup.find("div", {"class": "article-meta-item sp tijd"}).get_text()
        )

    def yields(self):
        return get_yields(
            self.soup.find("div", {"class": "article-meta-item sp aantal"}).get_text()
        )

    def image(self):
        return self.schema.image()

    def ingredients(self):
        ingredients = self.soup.find(
            "div", {"id": "clipboard-ingredients"}
        ).findChildren("li")

        return [normalize_string(i.get_text()) for i in ingredients]

    def instructions(self):
        # Old recipes are written in paragraphs, new ones are a list.
        paragraphs = self.soup.find("div", {"class": "preparation"}).find_all("p")
        listitems = self.soup.find("div", {"class": "preparation"}).find_all("li")
```